doc/cephadm/operations.rst

```diff
@@ -278,19 +278,31 @@ You can disable this health warning with::
 /etc/ceph/ceph.conf
 ===================
 
-Cephadm uses a minimized ``ceph.conf`` that only contains 
+Cephadm distributes a minimized ``ceph.conf`` that only contains 
 a minimal set of information to connect to the Ceph cluster.
 
-To update the configuration settings, use::
+To update the configuration settings, instead of manually editing
+the ``ceph.conf`` file, use the config database instead::
```

Comment:
```suggestion
Use the config database to update the configuration settings,
instead of manually editing the ``ceph.conf`` file::
```
minor reword to avoid using `instead` multiple times